package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "adal-angular": "1.0.15",
-    "ag-grid": "16.0.1",
+    "ag-grid": "18.1.2",
```

Comment:
I'm assuming you have confirmed that none of the breaking changes to ag-grid V17 or V18 from V16 affected this code.